docs/contributing/documentation/rfcs.md

```diff
@@ -3,6 +3,8 @@ title: Writing Request For Comment (RFC)
 description: Contributing RFCs to Powertools for AWS Lambda (Python)
 ---
 
+<!-- markdownlint-disable MD043 -->
+
```

Comment:
Is this related to the current issue? Can we remove this change before merging?

Reply:
Hello @rubenfonseca! This file was added in PR #3133. But it is breaking the local CI because there are no mandatory sections like "Getting started" and others, so I fixed it and we will update this file in another PR.